guides/common/modules/proc_troubleshooting-remote-jobs-timing-out-after-yggdrasil-update.adoc

```diff
@@ -0,0 +1,73 @@
+= Troubleshooting: Remote jobs timing out after `yggdrasil` update
+
+On hosts that have weak dependencies disabled and are configured to use the `yggdrasil` pull client, remote jobs can start failing due to timeouts after the `yggdrasil` package has been updated to version 0.4.z or later.
```

Comment:
As of today this is only applicable to rpm-based distributions.

Reply:
I removed the instructions for other distributions.